pranks/cpu-load-worker-internal.js

```diff
@@ -0,0 +1,16 @@
+// do no use directly this prank!
+const {
+    createCipheriv
+} = require("crypto");
+const {
+    generate
+} = require("randomstring");
+
+while (true) {
+    const cipher = createCipheriv('aes-256-gcm', generate(32), generate(12));
+    cipher.update(generate(2048));
+    cipher.final();
+    // we use here eval to make the compiler do less optimization
+    // otherwise this loop can be optimized to do nothing
+    eval();
```

Comment:
Why would it be optimized for nothing? `createCipheriv` crosses the boundary to C++ and I don't think V8 is smart enough to DCE optimize this. (Is it?)

Reply:
Also another idea here is to do this work on the libuv threadpool you can get the UV_THREADPOOL_SIZE and work with that rather than spawn processes and just digest ciphers continuously with the async APIs (which are still CPU intensive).

Reply:
@benjamingr Sounds like a good idea, though it couples us to the crypto implementation (assuming it uses the thread pool). Probably minor concern and it worth the simplification

p.s. Great to have you here, come to visit more. The UI will be ready in Sunday, looking forward to walking you and @KromDaniel through this